cursorless-talon/src/csv_overrides.py

```diff
@@ -151,7 +158,7 @@ def on_watch(path, flags):
                 handle_new_values=handle_new_values,
             )
 
-    fs.watch(str(file_path.parent), on_watch)
+    fs.watch(file_path.parent, on_watch)
```

Comment:
Type change in Talon. You can pass either a path instance or a string and both will work, but only a path object does not give type errors.